Update Dockerfile

shanzi opened the pull request:
It's time to migrate to Ubuntu 14.04 LTS now :D

I have tested it in my environment described below and it works well:

    OS version: Ubuntu 14.04
    Docker version: 1.0.1
    Go version: go1.2.1

Also, I think get `pip install -v` is better as it outputs more verbose log when installing with pip.

rutsky commented on master/contrib/Dockerfile:
```diff
@@ -64,13 +64,13 @@ from ubuntu:12.04
 run DEBIAN_FRONTEND=noninteractive apt-get install -y python-pip python-dev \
     supervisor git sudo ssh
-run pip install sqlalchemy==0.7.9 buildbot buildbot_slave
+run pip install -v sqlalchemy==0.7.9 buildbot buildbot_slave
```
Are you sure that verbose output of pip is needed?

shanzi replied:
When I am trying with the original docker file, it stops here a lot of time outputing nothing, so I think verbose with level one is better?

BTW: the `pip install` instructions in the traditional [firstrun](http://docs.buildbot.net/current/tutorial/firstrun.html) page installs `sqlalchemy==0.7.9`. Maybe we should updated here too?

shanzi replied:
If `-v` is not desired here, I will change it back to the origin.
Should I change `sqlalchemy==0.7.9` to `sqlalchemy==0.7.10`

rutsky replied:
@shanzi I just wanted to note, that your changes not only updates Ubuntu image version, but also changes verbosity. Initially I thought that you added `-v` in commit by mistake.
If `-v` doesn't add too much output and works as "progress bar" for end user, then it's ok.

rutsky commented:
@shanzi I can't comment about installing `sqlalchemy==0.7.9` or `sqlalchemy==0.7.10`, you need to ask other developers about it.

benallard commented:
I believe you shouldn't need to install sqlalchemy independently.

shanzi commented:
@benallard 
Are you meaning remove `sqlalchemy` from the `pip` command is ok? 

But both the instructions in the [First Run](http://docs.buildbot.net/current/tutorial/firstrun.html) page and the `Dockerfile` itself before my modification installs it by hand independently.

shanzi commented:
@benallard After reviewed the [setup.py](https://github.com/buildbot/buildbot/blob/master/master/setup.py#L375) of master and tested the Dockerfile without install sqlalchemy independently, I am convinced that `pip install sqlalchemy` is not necessary, so I dropped it. 

I think the [First Run](http://docs.buildbot.net/current/tutorial/firstrun.html) instructions may need update.

@rutsky  I also dropped the `-v` as I found the stopping and outputting nothing might be a temp network problem. The `pip`  without `-v` will output a proper amount of logs. :P

tardyp commented:
:+1:

djmitche commented:
The last run insructions have already been updated in the /latest/ docs (which are generated from master) -- http://docs.buildbot.net/latest/tutorial/firstrun.html

djmitche commented:
Nice work, thank you!
